config/level1/Boss.py3

```diff
@@ -157,19 +155,19 @@ while True:
         toggle = player  # to avoid boring draws we start our push phase differently
 
     # We select the item we want to search
-    for i in items:
-        if(i[0] == quests[0]):
-            goalPos = Vector(i[1], i[2])
+    for item in items:
+        if(item.name == quests[0]):
+            goalPos = item.position
 
     # If it's a push turn
-    if(turn_type == 0):
+    if(turnType == 0):
```

Comment:
no need of parentheses

Reply:
1bf3c5738d792dcbe2cf2f5a45f376365d993bde